data/human/pirates/pirate intro.txt

```diff
@@ -0,0 +1,732 @@
+	source "Darkstone"
+	destination "Greenrock"
+	to offer
+		has "event: slaver encounter"
+		has "tribute: Darkstone"
+	npc kill
+		personality nemesis
+		government "Pirate"
+		fleet "Small Southern Pirates" 4
+	on offer
+		conversation
+			`Darkstone looks somewhat less depressing on your next landing. The village near the landing pad, rather than being blacked-out, is well-lit. The town center is piled high with scrap, apparently looted from the gang you ran off during your last visit. They've certainly made good on their promise, given the amount of labor it must have taken to haul this much in so little time. In fact, the village is far more lively than you'd have expected - the old folk and children must have come out of hiding, and even the adults are more numerous than last count. From what you can see during landing, everybody is assembled in the center of town - and your ship isn't the only one on the landing pad. Visitors?`
+			choice
+				`	(Disembark and head to the village center.)`
+			`	The outskirts are abandoned. There's not a single hand tending the fields, or in the outlying buildings. They've all assembled at the town center.`
+			`	As it turns out, they're not there to render tribute unto you as you might have thought. No, the villagers are all being held at gunpoint by a gang of toughs in the center of the village green while a nasty-looking piece of work examines them one at a time, then directs them into one queue or another. Colonists in one are shackled together. Those in the other are made to lie flat with their hands behind their heads. Not a single colonist dares to look up; the spirit of defeat hangs too heavily over these broken souls.`
+			choice
+				`	"What's going on here?"`
+				`	(Stroll into the middle of town. Lock eyes with the leader of the slavers. Assume a gunfighter's stance.)`
+					goto duel
+				`	(Return to your ship. This is none of your business.)`
+					goto retreat
+				`	(Return to your ship... and start strafing the slavers.)`
+					goto attack
+			`	You're given an appraising look-over by the slaver chief. "... Yeah, you'd be too much trouble to sell. What'cha want, kid? We're busy here."`
+			choice
+				`	"These colonists are my property by right of conquest. If you're taking them, I expect to be fairly compensated."`
+				`	"Actually, I was just leaving."`
+					goto retreat
+				`	"Whoah, whoah, whoah, I'm cool with forcing people to give me stuff because I'm holding them at gunpoint, but forcing them into slavery? That's sick."`
+					goto challenge
+			`	The chief works his jaw for a moment, deep in thought. "Alright. You soundly thrashed the last bunch of bastards who were here, so I'm inclined to think we're best off doing business. Wholesale rate on these skins is ten thousand a body. 'course, that's with no shipping and handling - factor that in and we can afford to give you ehhh... Quarter million for the lot of 'em. You'll still have some left once we're done, mind."`
+			choice
```

Comment:
I think there should be an option to refuse the offer here as being too low, if we're going to take the 'terrible person' angle with this option.